docs/deployment.md

```diff
@@ -54,12 +54,13 @@ This command generates a config.yaml and a placeholder `samples.tsv` in the spec
 
 Here is what help looks like:
 ```bash 
+
 ##########################################################################################
 
 Welcome to
 ____ ____ ___  ____ _  _
 |__| [__  |__] |___ |\ |
-|  | ___] |    |___ | \|    v1.0.6
+|  | ___] |    |___ | \|    v1.0.6-dev-spikeins
```

Comment:
Might be best to leave out the version here so we don't have to constantly update it 

```suggestion
|  | ___] |    |___ | \|
```